assignments/callbacks.js

```diff
@@ -23,33 +23,78 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
   });
 
 */
-
-
-function getLength(arr, cb) {
+const list = [1, 2, 3, 'this'];
+function changeArr(arr, cb) {
+  return cb(arr);
+}
+function getLength(arr) {
```

Comment:
a little backwards but we talked about it in standup. I can tell you get the point of callbacks. get length would have been something like `function getLength(arr, cb) {return cb(arr.length}` and your cb could be anything, even console.log. a good rule of thumb is if you're having to rewrite the assignment (deleting all the cbs and moving them) it's probably a little off